modules/installation/nav-installation-guide.adoc

```diff
@@ -4,8 +4,13 @@ ifndef::backend-pdf[]
 * Installation Guide
 ** xref:install-intro.adoc[Introduction]
 ** Requirements
+ifeval::[{suma-content} == true]
 *** xref:general-requirements.adoc[General Requirements]
 *** xref:hardware-requirements.adoc[Hardware Requirements]
+endif::[]
+ifeval::[{uyuni-content} == true]
+*** xref:uyuni-install-requirements.adoc[Requirements]
+endif::[]
```

Comment:
I think you also need to split the network requirements doc below, or use ifs.

There are a lot of references to SLE15. I only noticed this by reviewing the PDF.

Reply:
ok, issue created. https://github.com/SUSE/doc-susemanager/issues/903